test/regression/2019-09-06-make-const-symbolic.c

```diff
@@ -0,0 +1,16 @@
+// REQUIRES: not-asan
```

Comment:
Can you add a comment about why ASan is not allowed?

Reply:
Probably just a copy-paste issue.
It should work without.